app/assets/stylesheets/style.scss

```diff
@@ -14,6 +14,13 @@ body {
 label {
   color: white;
 }
+a {
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background, color